### DISCUSSION TITLE
Repeater mark cloned row

### DISCUSSION THREAD

**Discussion by adrian-odysseystravel** [filamentphp/filament#13747] [Help]
### Package

Form builder

### Package Version

v3.2.85

### How can we help you?

I'm using a repeater with `clonable`, all working fine.
My question: is there any way of marking the newly cloned row? Like setting a color/bg. color/icon... I could not find anything in the docs or google.

Thank you!

**Comment by leandrocfe** [comment]
Do you mean that if a row is cloned, it will have a different color after the cloning action is triggered?

**Reply by adrian-odysseystravel** [reply]
Yes, exactly!
